start.yaml

```
@@ -18,6 +18,7 @@ max health: 15
 x: 0
 xp: 0
 y: 0
+map_zone: "Kathallion Plains"
```

You're in a dungeon

---

But before entering the dungeon, you're outside right?

---

Yes, but you enter the dungeon before you start playing

---

Also, I've been using spaces in my data file variables:
```suggestion
map zone: "Dungeon Entrance"
```

---

I'll fix that later.

---

I fixed that by commit aa666c7.